autofs-5.1.7 - remove unused variable from get_exports()

From: Ian Kent <raven@themaw.net>

Fix complier warning about unused variable entry in get_exports().

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG              |    1 +
 modules/lookup_hosts.c |    1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

--- autofs-5.1.4.orig/CHANGELOG
+++ autofs-5.1.4/CHANGELOG
@@ -49,6 +49,7 @@
 - move amd mounts removal into lib/mounts.c.
 - check for offset with no mount location.
 - remove mounts_mutex.
+- remove unused variable from get_exports().
 
 xx/xx/2018 autofs-5.1.5
 - fix flag file permission.
--- autofs-5.1.4.orig/modules/lookup_hosts.c
+++ autofs-5.1.4/modules/lookup_hosts.c
@@ -87,7 +87,6 @@ int lookup_read_master(struct master *ma
 static char *get_exports(struct autofs_point *ap, const char *host)
 {
 	char buf[MAX_ERR_BUF];
-	char entry[PATH_MAX + 1];
 	char *mapent;
 	struct exportinfo *exp, *this;
 	size_t hostlen = strlen(host);
